weaviate/data/crud_data.py

```diff
@@ -433,11 +435,14 @@ def get(
         with_vector: bool
             If True the `vector` property will be returned too,
             by default False
-        class_name : Optional[str], optional
+        class_name: Optional[str], optional
             The class name of the object with UUID `uuid`. Introduced in Weaviate version v1.14.0.
             STRONGLY recommended to set it with Weaviate >= 1.14.0. It will be required in future
             versions of Weaviate Server and Clients. Use None value ONLY for Weaviate < v1.14.0,
             by default None
+        limit: Optional[int], optional
+            The maximum number of data objects to return.
+            by default None
```

Comment:
maybe we should mention that `None` would result in Weaviate Server default value which is 100.

Reply:
good point, added :)